packages/flutterfire_ui/lib/src/i10n/lang/ar.dart

```diff
@@ -0,0 +1,264 @@
+    this.sendingSMSCodeText = 'جاري إرسال الرمز برسالة نصية...',
+    this.verifyingSMSCodeText = 'جاري التحقق من الرمز المرسل...',
+    this.enterSMSCodeText = 'أدخل الرمز المرسل',
+    this.emailIsRequiredErrorText = 'البريد الإلكتروني مطلوب',
+    this.isNotAValidEmailErrorText = 'رجاء قم بإدخال بريد إلكتروني صالح',
+    this.userNotFoundErrorText = 'هذا الحساب غير موجود',
+    this.emailTakenErrorText = 'الحساب الذي تحاول تسجيله موجود مسبقاً',
+    this.accessDisabledErrorText = 'تم تعطيل الوصول إلى هذا الحساب مؤقتًا',
+    this.wrongOrNoPasswordErrorText =
+        'كلمة المرور غير صالحة أو أن هذا المستخدم ليس لديه كلمة مرور',
+    this.signInText = 'تسجيل الدخول',
+    this.registerText = 'تسجيل جديد',
+    this.registerHintText = 'ليس لديك حساب مسبقا؟',
+    this.signInHintText = 'لديك حساب مسبقا؟',
+    this.signOutButtonText = 'تسجيل الخروج',
+    this.phoneInputLabel = 'رقم الهاتف',
+    this.phoneNumberInvalidErrorText = 'رقم الهاتف المدخل غير صالح',
+    this.phoneNumberIsRequiredErrorText = 'رقم الهاتف مطلوب',
+    this.profile = 'الملف الشخصي',
+    this.name = 'الاسم',
+    this.deleteAccount = 'حذف الحساب',
+    this.passwordIsRequiredErrorText = 'كلمة المرور مطلوبة',
+    this.confirmPasswordIsRequiredErrorText = 'قم بتأكيد كلمة مرورك',
+    this.confirmPasswordDoesNotMatchErrorText =
+        'كلمات المرور المدخلة غير متطابقة',
+    this.confirmPasswordInputLabel = 'تأكيد كلمة المرور',
+    this.forgotPasswordButtonLabel = 'نسيت كلمة المرور؟',
+    this.forgotPasswordViewTitle = 'استرجاع كلمة المرور المنسية',
+    this.resetPasswordButtonLabel = 'إعادة تعيين كلمة المرور',
+    this.verifyItsYouText = 'تحقق من هويتك',
+    this.differentMethodsSignInTitleText =
+        'استخدم إحدى الطرق التالية لتسجيل الدخول',
+    this.findProviderForEmailTitleText = 'أدخل بريدك الإلكتروني للمتابعة',
+    this.continueText = 'متابعة',
+    this.countryCode = 'رمز الدولة',
+    this.codeRequiredErrorText = 'رمز الدولة مطلوب',
+    this.invalidCountryCode = 'رمز الدولة هذا غير صالح',
+    this.chooseACountry = 'قم باختيار الدولة',
+    this.enableMoreSignInMethods = 'تفعيل المزيد من طرق تسجيل الدخول',
+    this.signInMethods = 'طرق تسجيل الدخول',
+    this.provideEmail = 'أدخل بريدك الإلكتروني وكلمة المرور',
+    this.goBackButtonLabel = 'عودة للخلف',
+    this.passwordResetEmailSentText =
+        'لقد أرسلنا إليك بريدًا إلكترونيًا يحتوي على رابط لإعادة تعيين كلمة المرور الخاصة بك. من فضلك تفقد بريدك الالكتروني',
+    this.forgotPasswordHintText =
+        'أدخل بريدك الإلكتروني وسنرسل لك رابطًا لإعادة تعيين كلمة مرورك',
+    this.emailLinkSignInButtonLabel = 'تسجيل الدخول باستخدام الرابط السحري',
+    this.signInWithEmailLinkViewTitleText =
+        'تسجيل الدخول باستخدام الرابط السحري',
+    this.signInWithEmailLinkSentText =
+        'أرسلنا لك رسال على البريد الإلكتروني تحتوي رابطاً سحرياً. تفقد صندوق رسائلك واضغط على الرابط لتسجيل الدخول',
+    this.sendLinkButtonLabel = 'أرسل الرابط السحري',
+    this.arrayLabel = 'array',
```

Comment:
@pr-Mais why did you ignore all labels below starting from this?

Reply:
I thought it's used internally in the code and doesn't show in the UI, will update it